src/qforte/adapters/molecule_adapters.py

```diff
@@ -235,6 +235,20 @@ def create_external_mol(**kwargs):
     with open(kwargs["filename"]) as f:
         external_data = json.load(f)
 
+    # extract symmetry information
+    point_group = external_data['point_group']['data']
```

Comment:
For backwards compatibility, can we assume that the point_group is C1 if not explicitly specified?